packages/notifier/test/test-notifier-adaptor.js

```diff
@@ -136,3 +140,211 @@ test('notifier adaptor - update from notifier fails', t => {
   const n = makeNotifierFromAsyncIterable(explodingStream);
   return observeNotifier(n, u);
 });
+
+// /////////////////////////////// makeNotifierFromSubscriber /////////////////////////////////
```

Comment:
please separate this into its own file. that will make it easier when doing watch tests for user to focus on what they want.